src/stores/blockly-store.ts

```diff
@@ -24,6 +25,13 @@ export const BlocklyStore = types
           // see https://github.com/microsoft/TypeScript/issues/31663
           (self[key] as any) = data[key] as any;
         });
+        // after we have loaded, set the current cml code to the initial. This will ensure that our
```

Comment:
`cml`?

Reply:
xml. Changed.

(cml was actually our old MW format, but I don't think my fingers were thinking of that.)